source/sixtrack.f90

```diff
@@ -233,11 +234,17 @@ subroutine daten
 !  READ FORT.3 HEADER
 ! ================================================================================================ !
 
+  call f_open(unit=3,file="fort.3",formatted=.true.,mode="r",err=fErr)
+  if(fErr) then
+    write(lout,"(a)") "INPUT> ERROR Could not open fort.2"
```

Comment:
it should be `write(lout,"(a)") "INPUT> ERROR Could not open fort.3"`

Reply:
Good catch. That's a typo. They put 2 and 3 too close to each other on the keyboard!